packages/playground/src/components/account/account-deposit/account-deposit.tsx

```diff
@@ -56,7 +56,7 @@ export class AccountDeposit {
     }
 
     this.updateAccount({
-      balance: web3.fromWei(parseFloat(this.amountDeposited), "ether")
+      balance: ethers.utils.formatEther(this.amountDeposited)
```

Comment:
Can we `import { formatEther } from "ethers/utils";` like in the the rest of the codebase?